src/taipy/gui/_gui_cli.py

```diff
@@ -28,7 +28,7 @@ def create_parser(cls):
             nargs="?",
             default="",
             const="",
-            help="The path to the WebApp to be used. The default is the webapp directory under gui in the Taipy GUI package directory.",
+            help="The path to the web app to be used. The default is the webapp directory under gui in the Taipy GUI package directory.",
```

Comment:
not web application here ?

Reply:
Here is how I used those terms: web app indicates the Taipy GUI web app, the one in "webapp/".
A web application is what a programmer could build, using Taipy GUI.